src/crawlee/proxy_configuration.py

```diff
@@ -124,19 +123,19 @@ async def new_proxy_info(
         if url is None:
             return None
 
-        # httpx.URL port field is None for default ports
-        default_ports = {'http': 80, 'https': 443}
-        port = url.port or default_ports.get(url.scheme)
-        if port is None:
+        if url.port is None:
```

Comment:
Just to be sure, so `yarl` returns e.g. `80` for `http://crawlee.dev`, correct?

Reply:
That's right.

80 for http://crawlee.dev
445 for http://crawlee.dev:445